<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="16" passed="9" failed="7" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-10T12:09:42 IST" name="API Test Suite" finished-at="2023-10-10T12:09:47 IST" duration-ms="5460">
    <groups>
    </groups>
    <test started-at="2023-10-10T12:09:42 IST" name="Add Restaurant Test" finished-at="2023-10-10T12:09:44 IST" duration-ms="2498">
      <class name="Automation.AddRestaurant">
        <test-method signature="Addresto()[pri:0, instance:Automation.AddRestaurant@4ebff610]" started-at="2023-10-10T12:09:42 IST" name="Addresto" finished-at="2023-10-10T12:09:44 IST" duration-ms="2419" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 expectations failed.
Expected status code <201> but was <400>.

Expected content-type "JSON" doesn't match actual content-type "".
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <201> but was <400>.

Expected content-type "JSON" doesn't match actual content-type "".

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:238)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:128)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:91)
at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:321)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:298)
at Automation.AddRestaurant.Addresto(AddRestaurant.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Addresto -->
      </class> <!-- Automation.AddRestaurant -->
    </test> <!-- Add Restaurant Test -->
    <test started-at="2023-10-10T12:09:45 IST" name="Get All Restaurant Test" finished-at="2023-10-10T12:09:45 IST" duration-ms="232">
      <class name="Automation.GetAllRestaurant">
        <test-method signature="GetAll()[pri:0, instance:Automation.GetAllRestaurant@5f354bcf]" started-at="2023-10-10T12:09:45 IST" name="GetAll" finished-at="2023-10-10T12:09:45 IST" duration-ms="222" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetAll -->
      </class> <!-- Automation.GetAllRestaurant -->
    </test> <!-- Get All Restaurant Test -->
    <test started-at="2023-10-10T12:09:45 IST" name="Approve restaurant" finished-at="2023-10-10T12:09:45 IST" duration-ms="116">
      <class name="Automation.Admin.ApproveRestaurant">
        <test-method signature="approveresto()[pri:0, instance:Automation.Admin.ApproveRestaurant@2f67b837]" started-at="2023-10-10T12:09:45 IST" name="approveresto" finished-at="2023-10-10T12:09:45 IST" duration-ms="100" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- approveresto -->
      </class> <!-- Automation.Admin.ApproveRestaurant -->
    </test> <!-- Approve restaurant -->
    <test started-at="2023-10-10T12:09:45 IST" name="Getallorders" finished-at="2023-10-10T12:09:45 IST" duration-ms="97">
      <class name="Automation.Admin.GetAllOrders">
        <test-method signature="getallorder()[pri:0, instance:Automation.Admin.GetAllOrders@1046d517]" started-at="2023-10-10T12:09:45 IST" name="getallorder" finished-at="2023-10-10T12:09:45 IST" duration-ms="62" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [malnad] but found [malnaddd]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [malnad] but found [malnaddd]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Automation.Admin.GetAllOrders.getallorder(GetAllOrders.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getallorder -->
      </class> <!-- Automation.Admin.GetAllOrders -->
    </test> <!-- Getallorders -->
    <test started-at="2023-10-10T12:09:45 IST" name="Getallrating" finished-at="2023-10-10T12:09:45 IST" duration-ms="71">
      <class name="Automation.Admin.GetAllRating">
        <test-method signature="getallrating()[pri:0, instance:Automation.Admin.GetAllRating@7a362b6b]" started-at="2023-10-10T12:09:45 IST" name="getallrating" finished-at="2023-10-10T12:09:45 IST" duration-ms="68" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getallrating -->
      </class> <!-- Automation.Admin.GetAllRating -->
    </test> <!-- Getallrating -->
    <test started-at="2023-10-10T12:09:45 IST" name="RejectRestaurant" finished-at="2023-10-10T12:09:46 IST" duration-ms="112">
      <class name="Automation.Admin.RejectRestaurant">
        <test-method signature="rejectresto()[pri:0, instance:Automation.Admin.RejectRestaurant@1c93f6e1]" started-at="2023-10-10T12:09:45 IST" name="rejectresto" finished-at="2023-10-10T12:09:46 IST" duration-ms="107" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rejectresto -->
      </class> <!-- Automation.Admin.RejectRestaurant -->
    </test> <!-- RejectRestaurant -->
    <test started-at="2023-10-10T12:09:46 IST" name="Add dish" finished-at="2023-10-10T12:09:46 IST" duration-ms="69">
      <class name="Automation.Restaurant.AddDish">
        <test-method signature="Adddish()[pri:0, instance:Automation.Restaurant.AddDish@363a52f]" started-at="2023-10-10T12:09:46 IST" name="Adddish" finished-at="2023-10-10T12:09:46 IST" duration-ms="62" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 expectations failed.
Expected status code <201> but was <400>.

Expected content-type "JSON" doesn't match actual content-type "".
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <201> but was <400>.

Expected content-type "JSON" doesn't match actual content-type "".

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:91)
at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:321)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:298)
at Automation.Restaurant.AddDish.Adddish(AddDish.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Adddish -->
      </class> <!-- Automation.Restaurant.AddDish -->
    </test> <!-- Add dish -->
    <test started-at="2023-10-10T12:09:46 IST" name="delete dish" finished-at="2023-10-10T12:09:46 IST" duration-ms="389">
      <class name="Automation.Restaurant.DeleteDish">
        <test-method signature="Deletedish()[pri:0, instance:Automation.Restaurant.DeleteDish@3adcc812]" started-at="2023-10-10T12:09:46 IST" name="Deletedish" finished-at="2023-10-10T12:09:46 IST" duration-ms="380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Deletedish -->
      </class> <!-- Automation.Restaurant.DeleteDish -->
    </test> <!-- delete dish -->
    <test started-at="2023-10-10T12:09:46 IST" name="disable dish" finished-at="2023-10-10T12:09:46 IST" duration-ms="81">
      <class name="Automation.Restaurant.Disabledish">
        <test-method signature="disabledih()[pri:0, instance:Automation.Restaurant.Disabledish@4b41dd5c]" started-at="2023-10-10T12:09:46 IST" name="disabledih" finished-at="2023-10-10T12:09:46 IST" duration-ms="74" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 expectations failed.
Expected status code <200> but was <500>.

Expected content-type "text/plain;charset=UTF-8" doesn't match actual content-type "application/json".
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <200> but was <500>.

Expected content-type "text/plain;charset=UTF-8" doesn't match actual content-type "application/json".

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:91)
at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:321)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:298)
at Automation.Restaurant.Disabledish.disabledih(Disabledish.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- disabledih -->
      </class> <!-- Automation.Restaurant.Disabledish -->
    </test> <!-- disable dish -->
    <test started-at="2023-10-10T12:09:46 IST" name="enable dish" finished-at="2023-10-10T12:09:46 IST" duration-ms="64">
      <class name="Automation.Restaurant.EnableDish">
        <test-method signature="enableDish()[pri:0, instance:Automation.Restaurant.EnableDish@1603cd68]" started-at="2023-10-10T12:09:46 IST" name="enableDish" finished-at="2023-10-10T12:09:46 IST" duration-ms="60" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 expectations failed.
Expected status code <200> but was <500>.

Expected content-type "text/plain;charset=UTF-8" doesn't match actual content-type "application/json".
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <200> but was <500>.

Expected content-type "text/plain;charset=UTF-8" doesn't match actual content-type "application/json".

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:91)
at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:321)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:298)
at Automation.Restaurant.EnableDish.enableDish(EnableDish.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enableDish -->
      </class> <!-- Automation.Restaurant.EnableDish -->
    </test> <!-- enable dish -->
    <test started-at="2023-10-10T12:09:46 IST" name=" dishbyid" finished-at="2023-10-10T12:09:46 IST" duration-ms="50">
      <class name="Automation.Restaurant.GetDishByid">
        <test-method signature="getDishbyid()[pri:0, instance:Automation.Restaurant.GetDishByid@4f0100a7]" started-at="2023-10-10T12:09:46 IST" name="getDishbyid" finished-at="2023-10-10T12:09:46 IST" duration-ms="46" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 expectations failed.
Expected status code <200> but was <404>.

Expected content-type "JSON" doesn't match actual content-type "text/plain;charset=UTF-8".
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <200> but was <404>.

Expected content-type "JSON" doesn't match actual content-type "text/plain;charset=UTF-8".

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:91)
at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:321)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:298)
at Automation.Restaurant.GetDishByid.getDishbyid(GetDishByid.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDishbyid -->
      </class> <!-- Automation.Restaurant.GetDishByid -->
    </test> <!--  dishbyid -->
    <test started-at="2023-10-10T12:09:47 IST" name=" updatedish" finished-at="2023-10-10T12:09:47 IST" duration-ms="82">
      <class name="Automation.Restaurant.UpdateDish">
        <test-method signature="updatedish()[pri:0, instance:Automation.Restaurant.UpdateDish@6581dc0a]" started-at="2023-10-10T12:09:47 IST" name="updatedish" finished-at="2023-10-10T12:09:47 IST" duration-ms="76" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[2 expectations failed.
Expected status code <200> but was <500>.

Expected content-type "JSON" doesn't match actual content-type "text/plain;charset=UTF-8".
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <200> but was <500>.

Expected content-type "JSON" doesn't match actual content-type "text/plain;charset=UTF-8".

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:91)
at io.restassured.internal.ValidatableResponseOptionsImpl.specification(ValidatableResponseOptionsImpl.java:321)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:298)
at Automation.Restaurant.UpdateDish.updatedish(UpdateDish.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatedish -->
      </class> <!-- Automation.Restaurant.UpdateDish -->
    </test> <!--  updatedish -->
    <test started-at="2023-10-10T12:09:47 IST" name=" get resataurant by location" finished-at="2023-10-10T12:09:47 IST" duration-ms="58">
      <class name="Automation.user.GetRestaurantByLoction">
        <test-method signature="getrestaurantbyloc()[pri:0, instance:Automation.user.GetRestaurantByLoction@32c4e8b2]" started-at="2023-10-10T12:09:47 IST" name="getrestaurantbyloc" finished-at="2023-10-10T12:09:47 IST" duration-ms="50" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getrestaurantbyloc -->
      </class> <!-- Automation.user.GetRestaurantByLoction -->
    </test> <!--  get resataurant by location -->
    <test started-at="2023-10-10T12:09:47 IST" name=" get resataurant by name" finished-at="2023-10-10T12:09:47 IST" duration-ms="73">
      <class name="Automation.user.GetRestaurantByName">
        <test-method signature="getbyname()[pri:0, instance:Automation.user.GetRestaurantByName@4f5991f6]" started-at="2023-10-10T12:09:47 IST" name="getbyname" finished-at="2023-10-10T12:09:47 IST" duration-ms="71" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getbyname -->
      </class> <!-- Automation.user.GetRestaurantByName -->
    </test> <!--  get resataurant by name -->
    <test started-at="2023-10-10T12:09:47 IST" name="give rating" finished-at="2023-10-10T12:09:47 IST" duration-ms="82">
      <class name="Automation.user.GiveRating">
        <test-method signature="giverating()[pri:0, instance:Automation.user.GiveRating@32c726ee]" started-at="2023-10-10T12:09:47 IST" name="giverating" finished-at="2023-10-10T12:09:47 IST" duration-ms="78" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- giverating -->
      </class> <!-- Automation.user.GiveRating -->
    </test> <!-- give rating -->
    <test started-at="2023-10-10T12:09:47 IST" name="place order" finished-at="2023-10-10T12:09:47 IST" duration-ms="97">
      <class name="Automation.user.PlaceOrder">
        <test-method signature="placeorder()[pri:0, instance:Automation.user.PlaceOrder@7f0d96f2]" started-at="2023-10-10T12:09:47 IST" name="placeorder" finished-at="2023-10-10T12:09:47 IST" duration-ms="80" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- placeorder -->
      </class> <!-- Automation.user.PlaceOrder -->
    </test> <!-- place order -->
  </suite> <!-- API Test Suite -->
</testng-results>
